test/_create_test_db.js

```diff
@@ -25,7 +25,7 @@ function create_tables (callback) {
 
 test('Create "users" table in test database', function (t) {
   create_tables(function (err, data) {
-    t.equal(data.command, 'INSERT', 'DB Table Created & Test Data Inserted');
+    t.equal(data[data.length-1].command, 'INSERT', 'DB Table Created & Test Data Inserted');
```

Comment:
data is now an array of the executed commands instead of just the last command - finds the last executed command to ensure it was insert.